models/evc.py

```diff
@@ -1410,6 +1410,45 @@ def handle_link_down(self):
 
         return success
 
+    @staticmethod
+    def get_interface_from_switch(uni: UNI, switches: dict) -> Interface:
+        """Get interface from switch by uni"""
+        switch = switches[uni.interface.switch.dpid]
+        interface = switch.interfaces[uni.interface.port_number]
+        return interface
+
+    def handle_topology_update(self, switches: dict):
+        """Handle changes in the topology"""
+        interface_a = self.get_interface_from_switch(self.uni_a, switches)
+        interface_z = self.get_interface_from_switch(self.uni_z, switches)
+        active = self.is_uni_interface_active(interface_a, interface_z)
+
+        if self.is_active() != active:
+            if active:
+                self.activate()
+            else:
+                self.deactivate()
+            self.sync()
+
+    def should_be_active(self, switches: dict):
```

Comment:
Can we have this renamed as `are_unis_active`? It'd reflected more closely what it actually does.

Reply:
Renamed